EIPS/eip-draft_Non-Fungible Token with usage rights_abbrev.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Non-Fungible Token with usage rights"
+status: Draft
+type: Standards Track
+author: KKimos <bol@zju.edu.cn>
```

Comment:
There must be at least one author with a GitHub account.
```suggestion
author: KKimos (@KKimos)
```
```suggestion
author: KKimos (@KKimos), KKimos <bol@zju.edu.cn>
```
Either of the above is acceptable.